keras_nlp/backend/config.py

```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
```

Comment:
Are all these new blank lines intentional? Version change in linter?

Reply:
Ooh this must be some find-replace shenanigans 😛

Reply:
Either way looks like it makes these files consistent with the rest so LGTM

Reply:
Yeah, we were not consistent, nor did our lint check. This was just a little script I wrote. No strong opinion on newline after copyright block or not.